Youhak/src/main/java/net/youhak/member/role/repository/RoleJpaRepository.java

```diff
@@ -0,0 +1,18 @@
+package net.youhak.member.role.repository;
+
+import net.youhak.member.role.domain.Role;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Modifying;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+
+public interface RoleJpaRepository extends JpaRepository<Role, Long> {
+
+    @Modifying
+    @Query("update Role r set r.isDeleted = true where r.id = :id")
```

Comment:
``` java
@Query("
    update Role r 
    set r.isDeleted = true 
    where r.id = :id")
```
쿼리에 있어서도 이렇게 바꾼느 것에 대해서 어떻게 생각하시나요?

Reply:
https://dejavuhyo.github.io/posts/java-multiline-string/
참고 자료입니다

Reply:
SQL 구문 별로 나누는 거 좋아요! 근데 저렇게 아래로 작성할거면 """ 써야하지 않나요?!

```suggestion
@Query("""
    update Role r 
    set r.isDeleted = true 
    where r.id = :id
""")
```